src/js/background.js

```diff
@@ -532,16 +530,22 @@ Badger.prototype = {
    */
   updateBadge: function(tabId){
     if (!this.showCounter()){
-      chrome.browserAction.setBadgeText({tabId: tabId, text: ""});
+      if(chrome.browserAction.setBadgeText){
+        chrome.browserAction.setBadgeText({tabId: tabId, text: ""});
+      }
       return;
     }
     var numBlocked = this.blockedTrackerCount(tabId);
-    if(numBlocked === 0){
-      chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#00cc00"});
-    } else {
-      chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#cc0000"});
+    if(chrome.browserAction.setBadgeBackgroundColor){
+      if(numBlocked === 0){
+        chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#00cc00"});
+      } else {
+        chrome.browserAction.setBadgeBackgroundColor({tabId: tabId, color: "#cc0000"});
+      }
+    }
+    if(chrome.browserAction.setBadgeText){
```

Comment:
The use case is to isolate browser specific code into its own region so developers can reason about it more easily.

For example, I'm imagining someone might be trying to fix some bug on chrome, if they come across this code, it might not be obvious that this intended for use on mobile.

Eventually, it would nice if browser specific code was isolated to independent files, then we could have a shim API. That is out of scope for this PR. But to make it easier to do in the future, it would be nice to have browser specific pieces clearly distinguished.